### PR TITLE
Dashboard Formatting

### LINKED ISSUE #18: Dashboard: Logs partial index
As an authenticated user,
On the dashboard page below the "Log a Symptom" section,
I see a section that displays my 10 most recently logged symptoms.

For each log,
I see the name of the symptom, the time and date at which I experienced the symptom, and any additional notes I included in the log.

This section also has a "See all logs" button (story #22)

**Implementation details:**
- [x] AR call will probably be something like `Log.all.order(:time).limit(10)`

### LINKED ISSUE #48: Dashboard UI Edits
- [x] sidebar with profile and med list should scroll with rest of page
- [x] partial log index on dashboard should be stacked vertically with one log per row.

*  look into using CSS flexbox for page arrangement which might make it easier for all these things to fit together in one page?

### PR DESCRIPTION
**Pull request by michaeljevans**
#### Description
- Add footer to credit openFDA for information provided
- Add flexbox to anchor footer to bottom of page (following all relevant content)
- Add `User` instance method #most_recent_logs to pull 10 most recent `Log`s ordered by time of occurrence
- Add button to dashboard to view all `Log`s - currently doesn't lead anywhere

#### Closes issue(s)
closes #18 - Dashboard: Logs partial index
closes #48 - Dashboard UI Edits

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -10,39 +10,47 @@
...
+                <a class="nav-link" <%= link_to "Dashboard", dashboard_path  %></a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  Symptom Logs
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                  <a class="dropdown-item" href="#">View All Symptom Logs</a>
+                  <div class="dropdown-divider"></div>
+                  <a class="dropdown-item" href="#">Something else here</a>
```
Comment (rrabinovitch) [inline app/views/layouts/application.html.erb:34]: we should finalize the nav bar and get rid of the placeholders if they're not being used

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -10,39 +10,47 @@
...
-        </button>
-        <!-- does this ^ do anything? -->
+    <div class="content">
+      <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #e3f2fd;"">
+        <% if current_user %>
+          <a class="navbar-brand" href="#">Main Menu</a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <!-- does this ^ do anything? -->
```
Comment (rrabinovitch) [inline app/views/layouts/application.html.erb:20]: this span can probably be deleted

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,39 +10,47 @@
...
+        <%= yield %>
+      </div>
     </div>
-    <!-- add footer w note acknowledging openFDA API -->
+    <footer class="footer">
+      <%= image_tag '/assets/open_fda_logo.png', alt: 'openFDA Logo', width: '160', height: '40' %>
+      <p>
+        Thanks and credit for the drug information present in this application belong to <a href="https://open.fda.gov/">openFDA</a>.<br>
+        See <a href="https://open.fda.gov/apis/">API documentation</a> for more details.
+      </p>
```
Comment (rrabinovitch) [inline app/views/layouts/application.html.erb:53]: good addition!

**Review by rrabinovitch — APPROVED** [review]
thanks for diving into the front end!
looking forward to checking it out in the browser

let's just make sure to also clean up the miscellaneous pieces in the nav bar that aren't being used / decide what to keep in there and make sure it actually links to things (the symptom logs button can be covered by my branch)

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ejdelsztejn) [reply app/views/layouts/application.html.erb:34]: Agreed.  I did wonder if maybe some things couldn't be moved to the navbar (ex: Edit Medication List, Add New Medication) to clean up the dashboard a bit, but that's not really necessary

File: app/models/user.rb [hunk 4]
```diff
@@ -25,4 +25,8 @@ def full_name
   def potential_symptoms
     Symptom.joins(:medications).pluck(:description).uniq
   end
+
+  def most_recent_logs
+    logs.order(when: :desc).limit(10)
```
Comment (ejdelsztejn) [inline app/models/user.rb:30]: Ahh, beautiful, clean AR 👍

**Review by ejdelsztejn — APPROVED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (gabichuelas) [reply app/views/layouts/application.html.erb:34]: I feel like putting 'edit medication list' and 'add new medication' in the navbar would feel really strange. They're actions directly tied to a user's profile/information not general website navigation...

Comment (rrabinovitch) [reply app/views/layouts/application.html.erb:34]: ^ agreed @gabichuelas

**Pull request merged**